tests/integration/components/public/event-map-test.js

```diff
@@ -0,0 +1,26 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('public/event-map', 'Integration | Component | public/event map', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  assert.expect(1);
+
+  this.render(hbs`{{public/event-map event=model.event}}`);
```

Comment:
You need to define a dummy event object with lat, long and location name and pass it inside for test.